pkgs/development/python-modules/undetected-chromedriver/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, selenium
+, requests
+, websockets
+}:
+
+buildPythonPackage rec {
```

```suggestion
buildPythonPackage {
```
you can remove the rec if we hardcode the repo name